docs/troubleshooting_guide.md

```diff
@@ -0,0 +1,358 @@
+### Troubleshooting Guide:
+This guide provides a searchable list of errors for a possible match/solution.
+
+
+#### 1. Missing `dg_name` in `cluster_config.json` manifestation:
+* If this is the error text:
```

Comment:
Not sure if this boilerplate adds a lot;  Is it self-evident that this is an error message?

Reply:
Fixed